### ISSUE TITLE
Separate ethical part from BSD part

### ISSUE THREAD

**Issue by jaredhirsch** [ContributorCovenant/hippocratic-license#21]
I think it might be useful if this hippocratic license could be tagged onto licenses other than BSD, as the question of copyright restrictions is separate from the question of ethical reuse.

I almost wonder if there could be a way of denoting "open source copyright-related license" + hippocratic license, some shorthand notation like `-H`, for example, BSD-H, or `(H)`, for example, MPL 2.0 (H)?

**Comment by typeswitch-dev** [comment]
This is of interest to me as well -- the ability to release software under other licenses, in particular more restrictive copyleft licenses, but with the No Harm clause (and the Services and Enforceability clauses) attached.

**Comment by elimisteve** [comment]
@6a68 Agreed! This would be great.

**Comment by CoralineAda** [comment]
BSD-Hippocratic 1.3 will most likely be released on Monday!

**Comment by decentral1se** [comment]
> in particular more restrictive copyleft licenses, but with the No Harm clause (and the Services and Enforceability clauses) attached.

Yes, absolutely! Off the top of my head, I'm thinking a GPLv3-H with the Freedom 0 dropped. No idea if that is feasible but I can dream :woman_dancing:. I basically just want to have protection against sublicensing which leads to undesirable usage or privatisation (see https://github.com/EthicalSource/hippocratic-license/issues/42)
